forgettable-frontend/src/Components/SearchBar/SearchBar.js

```diff
@@ -22,48 +22,77 @@ function SearchBar({placeholder, data}) {
     } else {
       setFilteredData(newFilter);
     }
+
+    if (hasAutocomplete === false){
+      // outputs data so that pages can use the filtered data
+    exportFilteredData(filteredData);
```

Comment:
I think for this one to be successfully output you need to add in:
 const [data, setData] = useState('');
  
  const exportFilteredData = (filteredData) => {
    setData(filteredData);
  }

then when declaring the SearchBar component you need to add exportFilteredData={exportFilteredData}